services/motion/motion_test.go

```diff
@@ -1542,3 +1369,81 @@ func validMoveOnGlobeRequest() MoveOnGlobeReq {
 		Extra: nil,
 	}
 }
+
+// ------------------------------------------------------------------------------------------------------------------------------------
+// The below functions exist for round trip testing purposes but since proto plans cannot be converted into a valid motionplan.Plan
+// these cannot exist in the main
```

Comment:
I don't know what `in the main` means

Reply:
I totally forgot to finish this. Thanks.